index.bs

```diff
@@ -237,6 +237,11 @@ It is also RECOMMENDED that the Picture-in-Picture window has a maximum and
 minimum size. For example, it could be restricted to be between a quarter and
 a half of one dimension of the screen.
 
+When {{pictureInPictureElement}} is set, the Picture-in-Picture window MUST
+be visible, even when the <a>Document</a> is not in focus or hidden. The user
+agent SHOULD provide a way for users to manually close the Picture-in-Picture
+window.
```

Comment:
I don't think this is quite right... or not stringent enough. I think what needs to happen is that:

* The pip window needs to be classified as a [=browsing context/top-level traversable=]. 
* The user agent MUST retain a  system visibility state if "visible" for the pip window - and should be above all other windows.

Reply:
Reading the [relevant parts of the HTML spec](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-and-session-history) I'm not sure if a PiP window should be a [top-level traversable](https://html.spec.whatwg.org/multipage/document-sequences.html#top-level-traversables), which makes it a [traversable navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#traversable-navigable), hence a [navigable](https://html.spec.whatwg.org/multipage/document-sequences.html#navigables) which "presents a Document to the user via its active session history entry". The reason I say this is that a PiP window doesn't present a Document (it's just a rendering surface for video) and doesn't affect session history AFAIK.

Reply:
Right, yes... it's not a navigable, but I think I think this should maybe run:

> Run the [focusing steps](https://html.spec.whatwg.org/#focusing-steps) with the Picture-in-Picture
window.

To focus it. It's still a Window and Document with some limited functionality.